build.sbt

```diff
@@ -1,20 +1,16 @@
 import Dependencies._
-inThisBuild(
```

Comment:
is this deprecated??? I liked it better :thinking:

Reply:
No, it's not deprecated. It looks a little like the old syntax, so I proposed to change it, but let's keep it, since I cannot change the template scalafix.g8 to use it only.

Reply:
I like the idea of grouping build-level settings together, just like you do for a project